taskmanager/features/bootstrap/FeatureContext.php

```diff
@@ -53,4 +86,44 @@ public function thereShouldBeTasksWithStatus($count, $status)
             $this->taskRegistry->getByStatus(Status::toDo())
         );
     }
+
+    /**
+     * @Given there is a task named :taskName with :status status
+     */
+    public function thereIsTaskNamed(string $taskName, Status $status)
```

Comment:
@mateuszmaslanka shouldn't we move all task related functions to task context?

Reply:
I'm not sure and I think we should discuss about this because I have some doubts...
As for now we have `FeatureContext` but what does `Feature` mean exactly? Why for example we have all scenarios for dealing with the user assignation in `FeatureContext` and not in some other specific context like `UserContext`?
I wonder if the context should be just one but the scenarios (features) should be separated to `task.feature`, `user.feature`, `status.feature`, etc...